<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PracticOOPVodaIoanClass</class>
 <widget class="QWidget" name="PracticOOPVodaIoanClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PracticOOPVodaIoan</string>
  </property>
  <widget class="QLineEdit" name="putDescription">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QListWidget" name="assignedList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>351</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="unassignedList">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>351</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>UNASSIGNED</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addVolunteer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Volunteer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="topThreeBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>430</y>
     <width>77</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TOP 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="asssignButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ASSIGN</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PracticOOPVodaIoan.qrc"/>
 </resources>
 <connections/>
</ui>
